src/main/java/ua/danit/final_project/services/EmployeeService.java

```diff
@@ -0,0 +1,13 @@
+package ua.danit.final_project.services;
+
+
+import ua.danit.final_project.entities.Employee;
+import ua.danit.final_project.entities.User;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWUu39d-W2AOP0Ck9j6j&open=AWUu39d-W2AOP0Ck9j6j)

[ISSUE_KEY]: AWUu39d-W2AOP0Ck9j6j